### PR TITLE
Add audit rules

### PR DESCRIPTION
**Pull request by DilipSequeira**

### CODE REVIEW

**Comment by DilipSequeira** [comment]
I'd like to find a better form of words that clarifies (a) if your assumption is found to be non-compliant, the assumption is that your submission will be moved to Open, but (b) the review committee has discretion to ignore compliance issues that are not significant.

File: inference_rules.adoc [hunk 1]
```diff
@@ -610,6 +610,12 @@ The following techniques are disallowed:
 * Techniques that only improve performance when there are identical
   samples in a query. For example, sorting samples in SSD.
 
+== Audit
+After submission, the submitters working group may request that an Available submission be audited. The audit will be funded by MLCommons with a limited time budget (in 0.7, at most 40 hours) and will be completed within 1 month of publication. An auditor will be selected by the review committee who has no conflict of interest with the audited submitter. 
+
```
Comment (ephremw) [inline inference_rules.adoc:615]: We discussed this at the meeting. Which category or categories (available, preview, and research) are subject to audit?

Comment (DilipSequeira) [reply inference_rules.adoc:615]: My understanding is that our conclusion was Available submissions only, as captured above in "may request that an Available submission be audited."

**Comment by christ1ne** [comment]
When we randomly choose to audit, is it focus on one submitter (scope include all the entries submitted) or one entry (one cell in the result table), or multiple-entries from one submitter? When we randomly choose, is it among all submitters or all result entries?

**Comment by DilipSequeira** [comment]
I suggest 

- it's among submitters, and the audit in principle applies to all entries. The WG, or the auditor, may choose to focus on the entries of most interest if there are such.

- it's not weighted by numbers of results entries.

**Comment by TheKanter** [comment]
I'd like to think very strongly about how we can minimize the engineering burden associated with submission (e.g., all audits, submitting, etc.).

**Comment by DilipSequeira** [comment]
> When we randomly choose to audit, is it focus on one submitter (scope include all the entries submitted) or one entry (one cell in the result table), or multiple-entries from one submitter? When we randomly choose, is it among all submitters or all result entries?

@christ1ne I think this is maybe most interesting for Intel, as you have a variety of disparate accelerators - so potentially much more of a burden for auditors and for you than e.g. someone who has to audit Nvidia's GPU submissions (which are all based on the same software stack and tools.) Do you have an opinion here? Maybe audit is one submitter/accelerator combination?

**Comment by christ1ne** [comment]
> > When we randomly choose to audit, is it focus on one submitter (scope include all the entries submitted) or one entry (one cell in the result table), or multiple-entries from one submitter? When we randomly choose, is it among all submitters or all result entries?
> 
> @christ1ne I think this is maybe most interesting for Intel, as you have a variety of disparate accelerators - so potentially much more of a burden for auditors and for you than e.g. someone who has to audit Nvidia's GPU submissions (which are all based on the same software stack and tools.) Do you have an opinion here? Maybe audit is one submitter/accelerator combination?

I think submitter/HW type combination as one audit unit makes sense.

**Comment by christ1ne** [comment]
replaced by https://github.com/mlperf/inference_policies/pull/177

**Pull request closed without merging**